assignment2/src/app/app.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  title = 'assignment2';
```

Comment:
Please define a meaningful name for the app.

Reply:
title = 'BOOK MANAGEMENT';